input-profile.ts

```diff
@@ -148,11 +148,11 @@ export class InputProfile extends Component {
         const stack: Object3D[] = [obj];
         while (stack.length > 0) {
             const currentObj = stack.pop();
```

Comment:
```suggestion
            const currentObj = stack.pop()!;
```

Should make the other ! obsolete :)